### ISSUE TITLE
Sun-setting alternative proposal.

### ISSUE THREAD

**Issue by rokyed** [atom/atom#25552]
Instead of archiving the project, why not transfer the ownership to the community?

Just think about it as a second life instead of a sunset.

Thank you very much.

TL;DR;
I love this editor, if it's not this editor then it's vim. It's amazing and I wouldn't change it for anything else, even after the sunset, I will still use it. Plan to use it forever. It's the only editor that you have 100% control over it. I don't like VS Code and no I will not switch to it because of personal beliefs. 

For the Atom community and team, you've made one of the greatest text editors out there. Nothing will ever beat it, not even vim which I also swear by. I have so much respect for you!!

I'm looking forward to forking this project if needed and continuing its life as it was destined to be.
TL;DR; end.

**Comment by israelidanny** [comment]
For what it is worth, this project is MIT licensed, and any community member can freely fork it & declare it the "Community Continuation". It'd be far too complex for the team to figure out who to "transfer" ownership to.

**Comment by softorangetech11** [comment]
> For what it is worth, this project is MIT licensed, and any community member can freely fork it & declare it the "Community Continuation". It'd be far too complex for the team to figure out who to "transfer" ownership to.

yeah, community fork, many other forks and stuff

**Comment by Verisimilitude11** [comment]
Good point.

**Comment by m1ga** [comment]
Be sure to check the Atom Community discord: https://discord.gg/J9fZJF4G  and https://github.com/AtomRebirth/atom
There is a lot going on already - including some new builds already :+1:

**Comment by Nobody-loses-like-trump** [comment]
But Why do you not like VS Code is it because it is a Microsoft Product? Atom is also indirectly a Microsoft Product. and you can disable Telemetry in VS Code if you want using the `"telemetry.telemetryLevel": "off",` setting (Even Atom also asks for Telemetry so that is not a valid reason anyway)

**Comment by m1ga** [comment]
@Nobody-loses-like-trump or use https://github.com/VSCodium/vscodium if you want to use it without tracking elements. But it is not about MS or telemetry. It's just a different editor and has more freedom to change the UI. Think that discussion has been out there since VSCode has been released 😄

**Comment by hermidalc** [comment]
I was a long time Atom user and switched to VSCode recently. Sorry to say but 90% of devs don't care about customizing the UI that much, it's a lot of huffing and puffing for nothing. Telemetry is a stupid argument just like @m1ga says there are ways to use VSCode without telemetry.

I've already been using VSCode extensions and features that OMG I wish Atom had I always needed them but didn't realize it was possible before. Once the vscode-R devs add a killer feature I've requested to make it work exactly like vscode-python wrt environments, which is yet another feature Atom never had and something even RStudio doesn't support, Atom will be in my memory box and I will feel nostalgia but in the end it's just a damn editor/IDE and I'm already super comfortable with VSCode.

**Comment by hermidalc** [comment]
> @hermidalc good

People should have something constructive to say and adding to the discussion otherwise it just looks really stupid.

**Comment by hermidalc** [comment]
If they https://github.com/atom-community resurrect Atom so that it has at least one of the killer features of VSCode and the project will evolve and grow, I will switch back. I used Atom almost everyday from 2017 until June 2022. I f---ing loved it. But I'm not a JS developer and have only been able to contribute with testing and issue submission.

One feature that Atom **needs** to have that VSCode has, is that you shouldn't have to be able to run Atom from a terminal in order to get it to work in an activated conda environment. VSCode you launch from your desktop and with each project folder you have open it will find all the python interpreters on your system and you can assign an specific interpreter (and therefore entire conda environment) to each project folder! It's exactly how many devs, including myself, develop. We need a different python, R, dependencies with their own versions/builds for each project folder (Github repo).

With Atom you can only have one conda environment in use per Atom instance (when you launch it from the terminal with that conda environment activated). This VSCode feature is really good.

**Comment by hermidalc** [comment]
Another feature that VSCode has which shows Atom just hasn’t evolved in a long time is VSCode has extensions for Snakemake language support and `snakefmt` Black formatting for Snakemake files. This is really useful for devs like me who use Snakemake a lot. Atom doesn’t even understand the Snakemake domain specific language built on top of python.

**Comment by Nobody-loses-like-trump** [comment]
> @Nobody-loses-like-trump or use https://github.com/VSCodium/vscodium if you want to use it without tracking elements. But it is not about MS or telemetry. It's just a different editor and has more freedom to change the UI. Think that discussion has been out there since VSCode has been released 😄

But What are the other tracking elements in VS Code apart from normal Telemetry which can be disabled with the `"telemetry.telemetryLevel": "off",` setting

**Comment by hermidalc** [comment]
> > @Nobody-loses-like-trump or use https://github.com/VSCodium/vscodium if you want to use it without tracking elements. But it is not about MS or telemetry. It's just a different editor and has more freedom to change the UI. Think that discussion has been out there since VSCode has been released 😄
> 
> But What are the other tracking elements in VS Code apart from normal Telemetry which can be disabled with the `"telemetry.telemetryLevel": "off",` setting

Then use VSCodium or use VSCode in a VM where you can turn off the internet in the VM when you don’t need it. All of this is starting to sound a lot like election conspiracy theories at this point, there are ways to not be tracked in VSCode if you don’t want to.

**Comment by w-henderson** [comment]
> > > @Nobody-loses-like-trump or use https://github.com/VSCodium/vscodium if you want to use it without tracking elements. But it is not about MS or telemetry. It's just a different editor and has more freedom to change the UI. Think that discussion has been out there since VSCode has been released 😄
> > 
> > But What are the other tracking elements in VS Code apart from normal Telemetry which can be disabled with the `"telemetry.telemetryLevel": "off",` setting
> 
> Then use VSCodium or use VSCode in a VM where you can turn off the internet in the VM when you don’t need it. All of this is starting to sound a lot like election conspiracy theories at this point, there are ways to not be tracked in VSCode if you don’t want to.
> 

This is getting a bit far fetched, on the one hand nobody wants to be tracked by Microsoft but on the other, you simply can't run your day-to-day editor in an isolated virtual environment with no internet access!

**Comment by marnen** [comment]
I would be happy to lend a hand in getting community ownership of Atom going. It's been my favorite editor for years, and no, I have no interest in switching to VS Code (I'd be more likely to go back to Sublime Text). It's OSS, and its viability shouldn't depend solely on GitHub's interest in maintaining it. 

Let's keep Atom viable! How can I help?

**Comment by marnen** [comment]
Possible duplicate of #25539.

**Comment by hermidalc** [comment]
> > > > @Nobody-loses-like-trump or use https://github.com/VSCodium/vscodium if you want to use it without tracking elements. But it is not about MS or telemetry. It's just a different editor and has more freedom to change the UI. Think that discussion has been out there since VSCode has been released 😄
> > > 
> > > 
> > > But What are the other tracking elements in VS Code apart from normal Telemetry which can be disabled with the `"telemetry.telemetryLevel": "off",` setting
> > 
> > 
> > Then use VSCodium or use VSCode in a VM where you can turn off the internet in the VM when you don’t need it. All of this is starting to sound a lot like election conspiracy theories at this point, there are ways to not be tracked in VSCode if you don’t want to.
> 
> This is getting a bit far fetched, on the one hand nobody wants to be tracked by Microsoft but on the other, you simply can't run your day-to-day editor in an isolated virtual environment with no internet access!

Then use VSCodium, it was another of the suggestions I put up. I’m just offering ideas to people who are really telemetry averse.

I’m not and a lot of devs aren’t, to me I always look at the big picture with all the pros and cons, which I think is more rational. Are the big features I am getting and allowing telemetry for worth it? Right now yes, I’ve listed at least three huge features that Atom is missing that VSCode has which make my dev life easier and better. I need an adept IDE now can’t wait for Atom to get back up to speed and maybe add these features in like a year or more. If they do I might come back but I don’t have the luxury to wait.

I doubt many of you complaining so hard about IDE telemetry have shut down all the telemetry on that thing in your hand called a smartphone. You need to install blocker apps and special mobile browser extension scripts not only in each of the mobile browsers you use on the phone but also block crazy in-app telemetry happening every second, let alone what you can’t block at all that Google and Apple are doing no matter what kind of blocker apps you use. 

That to me is WAY more privacy incursion than telemetry in my dev IDE! And all of us live with it every day! Not saying the state of tech is good. but we have to look at it rationally, there’s nothing right now that’s zero telemetry that isn’t a piece of shit phone.

**Comment by Ali-Aref** [comment]
Those who don't like `VsCode` switch to [Lunarvim](https://github.com/LunarVim/LunarVim), you will thank me later!  ;)

**Comment by stech11845** [comment]
> Those who don't like `VsCode` switch to [Lunarvim](https://github.com/LunarVim/LunarVim), you will thank me later! ;)

Vim is hard to use, + it is on the TERMINAL side of things, LunarVim seems to be terminal as well (No Testing).

**Comment by idleberg** [comment]
> Telemetry is a stupid argument just like @m1ga says there are ways to use VSCode without telemetry.

It's also a straw man argument:

- VSCode has telemetry built in
- Atom has telemetry built in
- Brackets has telemetry built in

Even worse, many third-party Atom packages have their own telemetry features (e.g. [atom-beautify](https://atom.io/packages/atom-beautify) or [build](https://atom.io/packages/build)) and most of them are not compliant with the General Data Protection Regulation (EU GDPR) or California Consumer Privacy Act (CCPA). All of Red Hat's own VSCode extensions have telemetry built in.

Anyway, let's stay on-topic. There are now two Atom forks on the rise:

- https://github.com/atom-community/
- https://github.com/pulsar-edit

Good luck to both of them!

**Comment by iamunadike** [comment]
Those who don’t like Ali switch to Christian religion you will thank me
later

**Comment by Xavifontv** [comment]
Hey, I have tried to join the discord community https://discord.gg/J9fZJF4G but it is not a valid invite... anyone has an invite for a discord dedicated to keeping atom alive?

**Comment by idleberg** [comment]
> anyone has an invite for a discord dedicated to keeping atom alive?

Discord invites:

- [atom-community](https://discord.gg/CREUNTqz)
- [pulsar-edit](https://discord.gg/udJsjRdm)
